docs/DeveloperGuide.md

```diff
@@ -722,13 +743,13 @@ The `tag` command allows user to tag a ModuleTag and Lessons to an existing cont
 
 <img src="images/TagActivityDiagram.svg" style="width:60%;margin:0 20%">
 <div style="width:60%;margin:0 20%;text-align:center">
-    <b>Figure 4.4.x</b> Sequence Diagram for a typical <code>TagCommand</code> execution 
+    <b>Figure 4.5</b> Sequence Diagram for a typical <code>TagCommand</code> execution 
 </div>
 <br>
 
 <img src="images/UntagActivityDiagram.svg" style="width:60%;margin:0 20%">
 <div style="width:60%;margin:0 20%;text-align:center">
-    <b>Figure 4.4.x</b> Sequence Diagram for a typical <code>UntagCommand</code> execution 
+    <b>Figure 4.6</b> Sequence Diagram for a typical <code>UntagCommand</code> execution 
```

Comment:
<img width="618" alt="image" src="https://user-images.githubusercontent.com/50812868/230291607-2b2158f8-d82d-41e7-bbd8-e7acff06a5c7.png">

I find that these [conditions] are not really clear. They do not give any context. E.g what tagged? What do you mean none tagged? AKA what is being tagged. A type level thing (module/group) or invalid tag?